src/metabase/driver/sql_jdbc/connection.clj

```diff
@@ -296,6 +296,19 @@
   [[jdbc-spec-binding [driver details]] & body]
   `(do-with-connection-spec-for-testing-connection ~driver ~details (^:once fn* [~jdbc-spec-binding] ~@body)))
 
+(defmacro with-unpooled-connection-spec
```

Comment:
This is identical to `with-connection-spec-for-testing-connection`. That macro does all of the helpful stuff of ensuring we have an open tunnel and the tunnel closes after we are done. It's perfect. But we don't want that name in the action execution code. And if they ever need to change in the future, we don't have to guess which ones are testing a connection and which are using a non-pooled connection on purpose.